### PR TITLE
Bug 1527107: Fix CVE-2019-5736

### PR DESCRIPTION
**Pull request by walac**
docker-ce >= 18.09.2 fixes this vulnerability. Unfortunatelly, docker
channel doesn't distribute this version for Ubuntu trusty. As trusty
support is expiring too, we update the Ubuntu version in the AMI to
bionic.

The default kernel for bionic in EC2 is the linux-image-aws, but it
doesn't come with V4L2, necessary for some tests. We then switch to the
generic kernel.

Notice bionic uses systemd, and we have to upgrade startup scripts to
cope with it. We had to modify some systemd scripts to break dependency
cycles.

### CODE REVIEW

File: deploy/packer/base/scripts/packages.sh [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 set -e -v
 
-DOCKER_VERSION=18.06.0~ce~3-0~ubuntu
-KERNEL_VER=4.4.0-1014-aws
-V4L2LOOPBACK_VERSION=0.10.0
+DOCKER_VERSION=18.03.1~ce~3-0~ubuntu
```
Comment (djmitche) [inline deploy/packer/base/scripts/packages.sh]: 18.03??

File: deploy/template/usr/local/bin/start-docker-worker [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+DOCKER_WORKER='node /home/ubuntu/docker_worker/src/bin/worker.js'
+DOCKER_WORKER_OPTS=
+META='http://170.254.169.254/latest/meta-data/'
```
Comment (djmitche) [inline deploy/template/usr/local/bin/start-docker-worker]: looks like this is unused?

File: deploy/packer/base/scripts/packages.sh (hunk 1 above)
Comment (walac) [reply deploy/packer/base/scripts/packages.sh]: It was a mistake, I am fixing it

File: deploy/template/usr/local/bin/start-docker-worker (hunk 2 above)
Comment (walac) [reply deploy/template/usr/local/bin/start-docker-worker]: Hrm, just to make it equal to its `upstart` counterpart.

Comment (djmitche) [reply deploy/template/usr/local/bin/start-docker-worker]: Does it need `export` then?

Comment (walac) [reply deploy/template/usr/local/bin/start-docker-worker]: It the original it hadn't

**Comment by djmitche** [comment]
I'm going to merge now to avoid conflicts, but wcosta will be deploying later.

**Pull request merged**